subsys/bootloader/cmake/sign.cmake

```diff
@@ -77,23 +77,36 @@ foreach (slot ${slots})
   set(signed_hex ${PROJECT_BINARY_DIR}/signed_by_b0_${slot}.hex)
   set(signed_bin ${PROJECT_BINARY_DIR}/signed_by_b0_${slot}.bin)
 
-  set(sign_depends ${PROJECT_BINARY_DIR}/${slot}.hex ${SIGN_KEY_FILE_DEPENDS})
+  set(sign_depends ${SIGN_KEY_FILE_DEPENDS})
 
   if (CONFIG_BUILD_S1_VARIANT AND NOT CONFIG_BOOTLOADER_MCUBOOT AND
       CONFIG_SPM AND ("${slot}" STREQUAL s1_image))
+    set(to_sign_incorrect_size ${PROJECT_BINARY_DIR}/s1_spm_app.hex)
```

Comment:
`incorrect` ?
could you find a better term, cause `incorrect` worries me.
Is it an offset, adjusted, reserved space for signing, or what does incorrect refers to.

Reply:
Changed to `unpatched`, does that look better?